CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 - More cleaning the body classname from the current displayname view @sneridagh
 - Make it possible to paste links, lists, b and i Elements into text-blocks @jackahl
+- remove title option from shown columns dropdown in contents view, as it is always shown anyways
```

Comment:
@jackahl please always start with an uppercase changelog. Also, please add your name at the end of the line.